modules/snowflake-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/snowflake/db/ast/CreateTable.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022-2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and
+ * limitations there under.
+ */
+
+package com.snowplowanalytics.snowplow.loader.snowflake.db.ast
+
+case class CreateTable(schema: String,
```

Comment:
I tried to simplify a bit. I think it is better wrap SQL statements with case classes though. Otherwise, it would kinda messy to having raw SQL directly where it is used.